scss/_patterns_form-help-text.scss

```
@@ -6,5 +6,10 @@
 
     color: $color-mid-dark;
     margin-top: $sp-unit * 2 - $spv-outer--scaleable; // negate the effect of scaling, as we don't want helper text position to change when the density multiplier changes
+
+    &.is-tick-element {
+      display: block;
+      margin-left: 2rem;
```

what does this 2rem represent? Width of checkbox box + standard spacing in between? If so, thijs would be better:
```suggestion
      margin-left: $sph-inner + $form-tick-box-size;
```

---

Ok. Thanks. I think it's the `display: block` what's breaking everything, and it's not even needed.